dpnp/dpnp_iface_searching.py

```diff
@@ -184,4 +184,14 @@ def where(condition, x=None, y=None):
 
     """
 
+    missing = (x is None, y is None).count(True)
+    if missing == 1:
+        raise ValueError("Must provide both 'x' and 'y' or neither.")
+    if missing == 2:
+        condition_desc = dpnp.get_dpnp_descriptor(condition, copy_when_nondefault_queue=False)
+        if condition_desc:
+            return dpnp_nonzero(condition_desc)
```

Comment:
wouldn't be simpler fust to call
```suggestion
        return dpnp.nonzero(condition)
```

Reply:
Done